html/pfappserver/lib/pfappserver/Form/Config/Network.pm

```diff
@@ -60,7 +60,7 @@ has_field 'ip_reserved' =>
   );
 has_field 'ip_assigned' =>
   (
-   type => 'TextArea',
+   type => 'ipAssigned',
```

Comment:
That is very :nail_care: but this should be called IPAssigned so that it follows the same standards as IPAddress and all its other IP related field friends that are in html/pfappserver/lib/pfappserver/Form/Field